### ISSUE TITLE
Not connecting to Spotify

### ISSUE THREAD

**Issue by csonner** [dvx/lofi#232]
Hello, 

Lofi is still giving me the blinking screen even with the Spotify App playing in the background. I opened the Debug Window and see the following error:
"Access token not set, ignoring fetch '/me/player?type=episode,track'
fetch @ renderer.bundle.js:2"

![Screenshot 2023-01-04 _ Connection to Spotify](https://user-images.githubusercontent.com/121966845/210611589-d7c017ce-f4d6-4927-921d-362f369ed745.jpg)

**Comment by stamoun** [comment]
Does this happen everytime you launch spotify?
Have you tried relaunching it?

Also, do you have a refresh/access token value set in the file `%appdata%\lofi\Settings` (use the run menu to open that, i.e. windows key + R)?

**Comment by csonner** [comment]
> Does this happen everytime you launch spotify? Have you tried relaunching it?
> 
> Also, do you have a refresh/access token value set in the file `%appdata%\lofi\Settings` (use the run menu to open that, i.e. windows key + R)?

1- This is my first time trying to use Lofi with Spotify
2- I've tried uninstalling & re-installing both Spotify and Lofi applications and that is not work. As well as running Lofi as admin

3 - When I follow your instruction above, there are two very long strings of characters associated with Refresh/Access Token but I'm not sure if mean anything.

""access_token":"BQDaXcdBZySzt2Pb9EZCRXR-ixwM6dAFf924npDw_RW_MZ8z5NghmQVGIACb_qkhFs2GZiqaJNMk7USFT_bnTBChLzLEwWuqXovT6l94hzHNdH4LHP_8NlHDVrK7zDCrPPrfd0iDsd23jv7By7JNvQVC7t31mWEo-E4Yf-JEBCDiAb4SrMqsG7JddWACcNJhYjtJm2A-OuqNSabxLzoea7v4fxJeA-M0lWSOggaV9hhh-axGvpeglw","refresh_token":"AQAh-50wtzfh6w9DLVHsSP_fK5f4_cHYdiXy-8ED00CvzJGO8Evj_Igi5hxRxxzVnHJSjYXKVxmOZaWZafRWCu3030bTZsqNEt8MzBkrqoEn7YiQFYqD8CM5DtCjrtIpoQQ1LA"}"

**Comment by stamoun** [comment]
If you see those tokens, it means you successfully logged in and allowed Lofi to access some of your Spotify data.

Not sure why nothing is displaying. Are you running Spotify alongside? Lofi can't run on it's own (for now)

**Comment by csonner** [comment]
Yes, I am running both simultaneously. I've tried both the desktop app & the web version of spotify.

**Comment by stamoun** [comment]
Weird, the way the code works, it should retrigger the track retrieval info every one second so you should see something once you've logged in.

What does the network tab look like?

**Comment by csonner** [comment]
![Screenshot 2023-01-04 Network](https://user-images.githubusercontent.com/121966845/210622523-85f730fa-ebea-4e67-8ba7-c0eb5c5ddcd8.jpg)

**Comment by stamoun** [comment]
So everything looks ok, this is the code that decides whether or not to display the data or the logo

```typescript
        {this.state.access_token ? (
          <Cover
            visualizationId={this.state.lofiSettings.visualization}
            settings={this.state.lofiSettings.window}
            side={this.state.window_side}
            lofi={this}
          />
        ) : (
          <Welcome lofi={this} />
        )}
```

I honestly have no clue why you only see the Spotify logo and not the art. You didn't even get a warning in the console this time around.

Can you try to close Lofi, delete the `%appdata%\lofi\Settings` file and relaunch Lofi? (You'll have to log in again)

**Comment by stamoun** [comment]
Most likely fixed in 2.0.1
